### PR TITLE
Remove DAML-on-X IndexServer and Reference Server

### PR DESCRIPTION
**Pull request by gerolf-da**
It is not needed anymore and can therefore be deleted.

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Add a line to the [release notes](https://github.com/digital-asset/daml/blob/master/unreleased.rst), if appropriate
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Comment by stefanobaghino-da** [comment]
−2,529
thumb_satisfaction-face-meme-generator-52804776

**Comment by gerolf-da** [comment]
@stefanobaghino-da: I bumped it up to -4k :)

@dajmaki: instead of the `api-server-damlonx` jar, this also publishes the sandbox jar as `com.daml.ledger:ledger-api-server` to maven. In the future this jar will not contain sandbox specifics, but for now it's identical.

**Comment by stefanobaghino-da** [comment]
> @stefanobaghino-da: I bumped it up to -4k :)

Double the satisfaction

File: docs/source/concepts/glossary.rst [hunk 1]
```diff
@@ -425,7 +425,7 @@ DAML ledgers provide various guarantees about what you can expect from it, all l
 
 When you're developing, you'll use `Sandbox <#sandbox>`__ as your ledger.
 
-If you want to run DAML on a storage mechanism of your choice, you can use the :doc:`/daml-integration-kit/index` to help you do that.
+If you want to run DAML on a storage mechanism of your choice, you can use one of the `supported platforms <https://daml.com>`__.
```
Comment (bame-da) [inline docs/source/concepts/glossary.rst]: If you would like to integrate DAML with a storage infrastructure not already in development (see `daml.com <https://daml.com`__), please get in touch on `Slack <https://damldriven.slack.com/sso/saml/start>`__ in channel #daml-contributors.

File: docs/source/index.rst [hunk 2]
```diff
@@ -94,7 +94,7 @@ DAML SDK documentation
    :caption: Experimental features
 
    experimental/warning
-   daml-integration-kit/index
+   tools/ledger-api-test-tool/index
```
Comment (bame-da) [inline docs/source/index.rst]: You are adding a new docs page, but don't mention it in the release notes. Are you intentionally silently changing integration kit to api-test-tool?

**Comment by dasormeter** [comment]
image

**Comment by dasormeter** [comment]
@gerolf-da did you remove all the daml integration kit documentation here?  i think we need to keep it, but in updated form

File: docs/source/concepts/glossary.rst (hunk 1 above)
Comment (gerolf-da) [reply docs/source/concepts/glossary.rst]: Thanks.

File: docs/source/index.rst (hunk 2 above)
Comment (gerolf-da) [reply docs/source/index.rst]: the ledger-api-test-tool was previously a sub-link of the daml integration kit. the test tool itself works fine and is supported by us. therefore I'm fine that it is still listed in the docs.

**Comment by gerolf-da** [comment]
> @gerolf-da did you remove all the daml integration kit documentation here? i think we need to keep it, but in updated form

@dasormeter: @dajmaki said that we can remove the docs completely.

**Pull request merged**